### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by Veekshith-Thoram**
image

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    --light-voilet: #cf9cf7;
+    --light-green: #b0f3d4;
+    --light-yellow: #fde68a;
+    --light-red: #fd8181;
+    --light-blue: #b0ddf5;
+    --blue: #0381F7;
+    --red: #DC2626;
+    --green: #059669;
+    --yellow: #f59e0b;
+}
```
Comment (AtulPant2704) [inline alert/alert.css]: You should have all of these colors in utility.css, not inside alert.css as you would need these colors at various pages.

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    --secondary-dark: #3F4652;
+    --light-voilet: #cf9cf7;
+    --light-green: #b0f3d4;
+    --light-yellow: #fde68a;
+    --light-red: #fd8181;
+    --light-blue: #b0ddf5;
+    --blue: #0381F7;
+    --red: #DC2626;
+    --green: #059669;
+    --yellow: #f59e0b;
```
Comment (AtulPant2704) [inline alert/alert.css]: Try to give names to the colors according to their use-case, not according to their color.

File: alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+}
+
+.line{
+    margin: 1rem;
+    padding: 0.1px;
+    background-color: gray;
+}
+
+iframe{
+    /* margin-left: 10%; */
```
Comment (AtulPant2704) [inline alert/alert.css]: You can remove this comment.

**Review by AtulPant2704 — CHANGES_REQUESTED** [review]
There is so much commented code in your PR, remove all of those comments and extract out your colors in a separate file.

File: alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+#primary-img{
+    width: 1.2rem;
+    height: 1.2rem;
+    /* margin-top: 1rem; */
+    margin-right: 3rem;
+}
+#success-img{
+    width: 1.2rem;
+    height: 1.2rem;
+    /* margin-top: 1rem; */
```
Comment (AnkurChunekar) [inline alert/alert.css]: Remove unnecessary comments.

File: alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+    /* text-align: center; */
+}
+
+.alerts{
+    /* text-align: center; */
+    /* display: flex; */
+    /* justify-content: center; */
+    /* flex-direction: row; */
+    position: relative;
+}
```
Comment (AnkurChunekar) [inline alert/alert.css:60]: add a last line at the end of the file to avoid getting this error

**Review by AnkurChunekar — CHANGES_REQUESTED** [review]

**Review by NagaSatyaGeeth — APPROVED** [review]

**Pull request merged**